BoyancyEngineTest/TOF.ino

```diff
@@ -12,13 +12,23 @@
     - add clock sync
     -
 */
-#define PRESSURE_SENSOR_REFRESH_RATE 5000 //in milliseconds // change back to 5000
 
-uint8_t reservoirDistanceTOF = 0;
+#define PRESSURE_SENSOR_REFRESH_RATE 200 //in milliseconds // change back to 5000
+#define TOFmin 160 
+#define TOFmax 20
+
+int reservoirDistanceTOF = 0;
+unsigned long previousReservoirDistanceTOF = 0;
+int reservoirFillPercentage = 0;
```

Comment:
Because arduino compiles files in alphabetical order, code won't compile untill this variable is palced in a file before pumptest. @johnreine any ideas about better ways to do this?